CMakeLists.txt

```diff
@@ -89,6 +87,11 @@ add_library(enet STATIC
     ${SOURCE_FILES}
 )
 
+target_include_directories(enet PUBLIC ${PROJECT_SOURCE_DIR}/include) # define the public header files path
+# consuming projects should be able to do #include "enet/enet.h" in their IDE (tested with CLion)
+
+file(COPY ${INCLUDE_FILES_PREFIX}/enet.h DESTINATION "include") # Copy the public header file to the build include directory for convenience
```

Comment:
This copies the `enet.h` file to wherever `${CMAKE_CURRENT_BINARY_DIR}/include` points to, but it **does it during the configuration step**, not when installing.